docs/sdk/authentication/session-sigs/intro.md

```diff
@@ -11,116 +11,63 @@ import FeedbackComponent from "@site/src/pages/feedback.md";
 `SessionSigs` are only available on Ethereum and are heavily in development, so things may change. Be sure to use the latest version of the Lit JS SDK.
 
 :::
+Session signatures are how the Lit network authenticates your connection, verifies your identity, and confirms your capabilities. 
 
-We refer to a session signature obtained from the user via session keys as a `SessionSig`.
+Generating session signatures is necessary whenever you want to request [Lit Abilities](https://v6-api-doc-lit-js-sdk.vercel.app/enums/types_src.LitAbility.html) (e.g. signing transactions with a particular PKP, executing a specified Lit Action) for your Lit Resources (e.g. PKPs, Lit Actions).
 
-`SessionSigs` are produced by a ed25519 keypair that is generated randomly on the browser and stored in local storage. The first step to producing `SessionSigs` is to first obtain an `AuthSig` through an authentication method like Google OAuth (example [here](https://github.com/LIT-Protocol/oauth-pkp-signup-example/blob/main/src/App.tsx#L398)). By specifying the session keypair's public key in the signature payload of the `AuthSig` - the `uri` field of the SIWE - users can choose which specific actions to delegate to the session keypair for operating upon certain resources.
+Session signatures are created with session keys, which are generated when you initiate a session through a request to a Lit network using the Lit SDK. Session keys are unique [`ed25519`](https://ed25519.cr.yp.to/) keypairs generated locally by the Lit SDK and are used to sign all requests to the Lit Network during the current session. 
 
-The session keypair is used to sign all requests to the Lit Nodes, and the user's `AuthSig` is sent along with the request, attached as a "capability" to the session signature. Each node in the Lit Network receives a unique signature for each request, and can verify that the user owns the wallet address that signed the capability.
+While session signatures facilitate ongoing communication during a session, an `AuthSig` (Authentication Signature) is used to verify your identity and authorization to the Lit Nodes.
 
-## Capability Objects
+An `AuthSig` is an [ERC-5573](https://eips.ethereum.org/EIPS/eip-5573) Sign-In with Ethereum Capabilities message that specifies the Lit Resources and Lit Abilities of the session.
 
-Session signatures work by having scoped capabilities be granted to session keys by an inner `AuthSig`. The capability object is a [SIWE ReCap](https://eips.ethereum.org/EIPS/eip-5573) object.
+The `AuthSig` allows Lit nodes to verify your authorization for requested actions, such as decrypting data, signing transactions with a PKP, or transferring PKP ownership. This ensures that only authorized users can perform specific actions within the Lit network.
+When you make a request, each Lit Node checks your `AuthSig` to confirm that your request aligns with the capabilities you previously defined. 
```

Comment:
Consider:

An AuthSig is an [ERC-5573](https://eips.ethereum.org/EIPS/eip-5573) Sign-In with Ethereum Capabilities message. It defines the specific Lit Resources and Lit Abilities authorized for your session.

The AuthSig enables the Lit nodes to verify your authorization to perform various actions within the Lit network. These actions may include decrypting data, signing transactions with a PKP, or executing Lit Actions.

When you submit a request to the Lit network, each node checks the AuthSig contained within your Session Signatures to ensure that your request aligns with the capabilities granted for your session. This validation process guarantees that only authorized users can perform specific actions using the Lit Resources defined by the AuthSig, maintaining the security and integrity of the Lit network.

Reply:
Blended them:

> An `AuthSig` is an [ERC-5573](https://eips.ethereum.org/EIPS/eip-5573) Sign-In with Ethereum Capabilities message. It specifies the authorized Lit Resources and Lit Abilities of the session.
> 
> The `AuthSig` allows Lit nodes to verify your authorization to perform requested actions, such as decrypting data, signing transactions with a PKP, or transferring PKP ownership. When you make a request, each Lit node checks your `AuthSig` to confirm that your request aligns with the capabilities you previously defined. This ensures that only authorized users can perform specific actions within the Lit network. This authentication system maintains the security and integrity of the Lit network.
> 
> For detailed explanations of this setup, please refer to our [Security Considerations](../security.md) page.

Reply:
> or transferring PKP ownership

This is done by transferring the underlying NFT which doesn't necessarily require a Session Sig. I want replace this with `or executing Lit Actions`

Reply:
Changed!